### PR TITLE
Release v0.8.0

### PR DESCRIPTION
**Pull request by thanethomson**
Supersedes #54.

:book: [CHANGELOG](https://github.com/cometbft/cometbft-db/blob/release/v0.8.0/CHANGELOG.md)

### CODE REVIEW

**Review by andynog — COMMENTED** [review]
Changes look good to me from a release perspective

**Review by cason — APPROVED** [review]
Legit.

File: .changelog/v0.8.0/summary.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+*Apr 25, 2023*
+
+This release bumps the supported version of RocksDB, which requires cometbft-db
+RocksDB users to update their builds (and hence requires a "major" release, but
+does not introduce any other breaking changes). Special thanks to @yihuang for
+this update!
+
+While the minimum supported version of the Go compiler was bumped to 1.19, no
+1.19-specific code changes were introduced and this should, therefore, still be
+able to be compiled with earlier versions of Go. It is, however, recommended to
```
Comment (sergio-mena) [inline .changelog/v0.8.0/summary.md:10]: Would it make sense to add a sentence here saying that 1.19 is the version used by the core team for all our testing? (CI, e2e, pre-release QA)

**Review by sergio-mena — APPROVED** [review]

File: .changelog/v0.8.0/summary.md (hunk 1 above)
Comment (thanethomson) [reply .changelog/v0.8.0/summary.md:10]: Is that not covered in the README by stating the minimum Go version? https://github.com/cometbft/cometbft-db#minimum-go-version

**Pull request merged**